doc/user/layouts/partials/create-source/connector/s3/syntax.html

```diff
@@ -0,0 +1,3 @@
+bucket_name | The name of the bucket to scan through.
```

Comment:
should be italicized for consistency, but realizing now maybe nothing should be italicized!

Reply:
oh interesting yeah I thought italics were for types, not parameters. I'll italicize it and we can change it uniformly.